CMakeLists.txt

```diff
@@ -39,8 +41,8 @@ set_target_properties(fbjni PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS ON)
 target_compile_options(fbjni PRIVATE ${FBJNI_COMPILE_OPTIONS})
 target_compile_options(fbjni PRIVATE -DBUILDING_FBJNI)
 
-target_include_directories(fbjni PUBLIC
-  cxx
+target_include_directories(fbjni BEFORE
+  PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/cxx> 
 )
 
 if (NOT ANDROID_ABI)
```

Comment:
Should we also fix the usage of `target_include_directories` that is related to `JAVA_HOME` in the following lines? Actually, I don't think they should be in the installed configuration file.

Reply:
I'm honestly not sure. ☹️ This is all a bit magic to me, so that may be a question for @dreiss.